mod_modular_vanilla/hooks/items/item.nut

```diff
@@ -41,8 +98,42 @@
 
 			this.setValuesBeforeRandomize(this.m.BaseItemScript == null ? null : ::new(this.m.BaseItemScript));
 			this.randomizeValues();
+
+			// MV: Added
+			// Part of MV_Variant framework for items.
+			// Roll a chance for using one of the variants provided using the MV framework.
+			if (!::MSU.Serialization.isLoading() && ::Math.rand(1, 100) <= this.m.MV_VariantChance)
```

Comment:
This might break with your persistant character mod, as `isLoading` is only true during loading of world map, not any kind of deserialization.

So using that mod, the color coding of your persistant characters would change each time they appear/wander

I dont have an easy solution, other than to fix this in MSU, like you suggested in discord:
> I see. So, I suppose we need to set isLoading to true during deserialization of serde emulators in MSU.

This is mainly a comment showcasing another bug resulting from reliance on `isLoading`